README.FreeBSD

```diff
@@ -159,4 +159,4 @@ Task to do:
 	details to figure out
 
  -	Design a vitual disk implementation that can be used with behyve and 
-	attached to an RBD image. 
+	attached to a RBD image. 
```

Comment:
These are actually correct -- ```an``` should be used instead of ```a``` since the "R" sound in the acronym "RBD" starts with an /a/ sound.

Reply:
... and it's general advisable to not combine unrelated commits within a single PR.